core/src/test/java/tech/tablesaw/io/csv/CsvReaderTest.java

```diff
@@ -161,6 +161,17 @@ void allowDuplicateColumnNames() throws IOException {
     assertEquals("Col1-2", dupes.columnNames().get(1));
   }
 
+  @Test
+  void allowDuplicateColumnNamesInsensitive() throws IOException {
+    final Reader reader2 =
+        new StringReader(
+            "Col1" + COMMA + "col1" + LINE_END + "\"first\"" + COMMA + "second" + LINE_END);
```

Comment:
We could probably make this a tad simpler 
```suggestion
            "Col1" + COMMA + "col1" + LINE_END + "first" + COMMA + "second" + LINE_END);
```

Reply:
Sure, I took that from the test above. Done